game/src/main/kotlin/content/area/misthalin/varrock/BrassKey.kts

```diff
@@ -0,0 +1,17 @@
+package content.area.misthalin.varrock
+
+import content.entity.obj.door.enterDoor
+import content.entity.sound.sound
+import world.gregs.voidps.engine.client.message
+import world.gregs.voidps.engine.entity.obj.objectOperate
+import world.gregs.voidps.engine.inv.inventory
+
+objectOperate("Open", "door_27_closed") {
```

Comment:
Let's rename door_27 to something more edgeville related (don't forget to rename the _open one too)